File: src/main/java/nl/hsleiden/model/Order.java
package nl.hsleiden.model;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.ArrayList;
import java.util.Date;

public class Order {

	@JsonProperty
	Account account;

	@JsonProperty
	String paymentBank;

	@JsonProperty
	String delivery;

	@JsonProperty
	ArrayList<CartRowProduct> orderRows;

	@JsonProperty
	Date orderDate;

	@JsonProperty
	String orderState;





}

File: Webshop/src/app/order/order.ts
import {CartRowProduct} from "../cart/cart-row-product";

export class Order {

  constructor() {

  }

  id:number;
  accountid:number;
  paymentBank: String;
  delivery: String;

  orderRows:CartRowProduct[];
  orderDate:Date;
  orderState:String;

}
